WooCommerce/WooCommerceTests/Tools/PriceInputFormatterTests.swift

```diff
@@ -147,6 +147,22 @@ final class PriceInputFormatterTests: XCTestCase {
         XCTAssertEqual(formatter.format(input: input), "-12.34")
     }
 
+    func test_formatting_when_decimal_separator_has_spaces() {
+        // Given store decimal separator has spaces
+        let currencySettings = CurrencySettings(currencyCode: .USD,
+                                                currencyPosition: .leftSpace,
+                                                thousandSeparator: ",",
+                                                decimalSeparator: ".   ",
+                                                numberOfDecimals: 2)
+        let formatter = PriceInputFormatter(currencySettings: currencySettings)
+
+        // When
+        let input = "189.20"
+
+        // Then formatting ignores spaces in store decimal separator settings
+        XCTAssertEqual(formatter.format(input: input), "189.20")
```

Comment:
We're not testing for space sanitization here, are we? If the `input` simulates what the merchant introduces in the textfield, and then this input is sanitized within `formatter.format(value)`, shouldn't we pass the value in as malformed to the formatter? (The test still succeeds)


```suggestion
        // When
        let input = "189.           20"

        // Then formatting ignores spaces in store decimal separator settings
        XCTAssertEqual(formatter.format(input: input), "189.20")
```